### PR TITLE
Delta CRL note

### PR DESCRIPTION
**Pull request by NagaCSC**
add note for Delta CRL

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ To resolve this issue, the CRL distribution point must be a location that is acc
 
 If your CRL distribution point does not list an HTTP distribution point, then you need to reconfigure the issuing certificate authority to include an HTTP CRL distribution point, preferably first in the list of distribution points.
 
+Note:  If your CA has both Base and Delta CRL published. please make sure. you have included publishing the delta CRL in the http path. Include web server to fetch delta crl by allowing doubleescaping in the (IIS) web server.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
Note:  If your CA has published both the Base and the Delta CRL, please make sure you have included publishing the Delta CRL in the HTTP path. Include web server to fetch the Delta CRL by allowing double escaping in the (IIS) web server.
```
This is just a preliminary suggestion of grammar corrections. @JohanFreelancer9 might have additional suggestions for other ways to phrase the contents of these sentences.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 2]
```diff
@@ -58,6 +58,8 @@ To resolve this issue, the CRL distribution point must be a location that is acc
 
 If your CRL distribution point does not list an HTTP distribution point, then you need to reconfigure the issuing certificate authority to include an HTTP CRL distribution point, preferably first in the list of distribution points.
 
+Note:  If your CA has published both the Base and the Delta CRL, please make sure you have included publishing the Delta CRL in the HTTP path. Include web server to fetch the Delta CRL by allowing double escaping in the (IIS) web server.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
> [!NOTE]
> If your CA has published both the Base and the Delta CRL, please make sure you have included publishing the Delta CRL in the HTTP path. Include web server to fetch the Delta CRL by allowing double escaping in the (IIS) web server.
```

**Comment by illfated** [comment]
@NagaCSC : Cosmetic detail, but a useful visual detail for other users: if you could correct the typo in your PR title, it would be easier to associate this PR with its content.

Delat CRL note ==> **Delta CRL note**

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 3]
```diff
@@ -151,6 +154,7 @@ These procedures configure NTFS and share permissions on the web server to allow
 ![CDP Share Permissions](images/aadj/cdp-share-permissions.png)
 9. In the **Advanced Sharing** dialog box, click **OK**.
 
+
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]: Cosmetic again, but I don't think we need this extra blank line.
```suggestion
```

**Comment by illfated** [comment]
#sign-off

**Pull request merged**